docs/Using-the-dataset-validator.md

```diff
@@ -414,3 +414,78 @@ As an example, the command for the "Example 1" listed above incorporating the `-
+-d ROOT_DIRECTORY, --root-directory ROOT_DIRECTORY
+                      Full path to directory with all studies that should be
+                      validated
+-l LIST_OF_STUDIES, --list-of-studies LIST_OF_STUDIES
+                      List with paths of studies which should be validated
+-html HTML_FOLDER, --html-folder HTML_FOLDER
+                      Path to folder for output HTML reports
+-u URL_SERVER, --url_server URL_SERVER
+                      URL to cBioPortal server. You can set this if your URL
+                      is not http://localhost/cbioportal
+-p PORTAL_INFO_DIR, --portal_info_dir PORTAL_INFO_DIR
+                      Path to a directory of cBioPortal info files to be
+                      used instead of contacting a server
+-n, --no_portal_checks
+                      Skip tests requiring information from the cBioPortal
+                      installation
+-P PORTAL_PROPERTIES, --portal_properties PORTAL_PROPERTIES
+                      portal.properties file path (default: assumed hg19)
+```
+
+Parameters `--url_server`, `--portal_info_dir`, `--no_portal_checks` and `--portal_properties` are equal to the parameters with the same name in `validateData.py`. The script will save a log file with validation output (`log-validate-studies.txt`) and output the validation status from the input studies:
+
+```console
+=== Validating study ../../../test/scripts/test_data/study_es_0
+Result: VALID (WITH WARNINGS)
+
+=== Validating study ../../../test/scripts/test_data/study_es_1
+Result: INVALID
+
+=== Validating study ../../../test/scripts/test_data/study_es_invalid
+directory cannot be found: ../../../test/scripts/test_data/study_es_invalid
+Result: INVALID (PROBLEMS OCCURRED)
+
+```
+
+### Example 1
+The script can be run for all studies in a certain directory by using the `--root-directory` parameter. The script will append each folder in the root directory to the study list to validate:
+```bash
+./validateStudies.py -d ../../../test/scripts/test_data/
+```
+
+### Example 2
+The script can also be run for specific studies by using the `--list-of-studies` parameter. The paths to the different studies can be defined and seperated by a comma:
+```bash
+./validateStudies.py -l ../../../test/scripts/test_data/study_es_0,../../../test/scripts/test_data/study_es_1
+```
+
+### Example 3
+The script can also be run on specific studies in a certain directory by combining the `--root-directory` and `--list-of-studies` parameter:
+```bash
+./validateStudies.py -d ../../../test/scripts/test_data/ -l study_es_0,study_es_1
+```
+
+### Example 4
+When HTML validation reports are desired for each of the studies validated an output folder for these HTML files can be specified. This folder does not have to exist, the script can create the folder. The HTML validation reports will get the following name: `<study_name>-validation.html`. To create HTML validation reports for each study the `--html-folder` parameter needs to be defined:
```

Comment:
Please make it a bit shorter:

"When HTML validation reports are desired, an output folder for these HTML files can be specified."